internal/project/framework/internals/middleware/latency.go

```diff
@@ -1,13 +1,14 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package middleware
 
 import (
 	"fmt"
 	"net/http"
 	"time"
 
-	"{{ .PkgPath }}/framework/internals/async"
-	"{{ .PkgPath }}/framework/internals/latency"
+	/**#bean*/
+	"demo/framework/internals/async" /*#bean.replace("{{ .PkgPath }}/framework/internals/async")**/
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines